src/development/tools/devtools/_profiler.md

```diff
@@ -1,101 +1,192 @@
-<dt markdown="1">**Self time**</dt>
-<dd>Time the method spent executing only its own code.</dd>
-<dt markdown="1">**Method**</dt>
-<dd>Name of the called method.</dd>
-<dt markdown="1">**Source**</dt>
-<dd>File path for the method call site.</dd>
+<dt markdown="1">**Total time**
+</dt>
+<dd>Time that a method spent executing its own code as well as
+    the code for any methods it called.
+</dd>
+
+<dt markdown="1">**Self time**
+</dt>
+<dd>Time the method spent executing only its own code.
+</dd>
 </dl>
 
-![Screenshot of a call tree table]({{site.url}}/assets/images/docs/tools/devtools/cpu_profiler_call_tree.png){:width="100%"}
+### Method table
 
-### Bottom up
+The method table provides CPU statistics for each method
+contained in a CPU profile. In the table on the left,
+all available methods are listed with their **total** and
+**self** time. 
 
-The bottom up view shows the method trace for the CPU profile but,
-as the name suggests, it's a bottom-up representation of the profile.
-This means that each top-level method in the table is actually the
-last method in the call stack for a given CPU sample (in other words,
-it's the leaf node for the sample).
+**Total** time is the combined time that a method spent
+**anywhere** on the call stack, or in other words,
+the time a method spent executing its own code and
+any code for methods that it called.
 
-In this table, a method can be expanded to show its _callers_.
+**Self** time is the combined time that a method spent
+on top of the call stack, or in other words,
+the time a method spent executing only its own code.
 
-<dl markdown="1">
-<dt markdown="1">**Total time**</dt>
-<dd markdown="1">Time the method spent executing its own code
-    as well as the code for its callee.
+![Screenshot of a call tree table]({{site.url}}/assets/images/docs/tools/devtools/method-table.png)
+See the [Guidelines](#guidelines) section below to learn how
+to enable the blue and green vertical lines seen in this image.
+
+Selecting a method from the table on the left shows
+the call graph for that method. The call graph shows
+a method’s callers and callees and their respective
+caller / callee percentages.
+
+### Flame chart
+
+The flame chart view is a graphical representation of
+the [Call tree](#call-tree). This is a top-down view
+of a CPU profile, so in this chart,
+the top-most method calls the one below it.
+The width of each flame chart element represents the
+amount of time that a method spent on the call stack.
+
+Like the Call tree, this view is useful for identifying
+expensive paths in a CPU profile.
+
+![Screenshot of a flame chart]({{site.url}}/assets/images/docs/tools/devtools/cpu-flame-chart.png)
+
+The help menu, which can be opened by clicking the `?` icon
+next to the search bar, provides information about how to
+navigate and zoom within the chart and a color-coded legend.
+![Screenshot of flame chart help]({{site.url}}/assets/images/docs/tools/devtools/flame-chart-help.png)
```

Comment:
on the staged site, this image feels a bit too large. Can set a smaller max size?

Reply:
Not globally, but certainly on a per image basis.